code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1209,7 +1201,6 @@ so that different stomachs can handle things in different ways VB*/
 	if(HAS_TRAIT(src, TRAIT_XRAY_VISION))
 		sight |= (SEE_TURFS|SEE_MOBS|SEE_OBJS)
 		see_in_dark = max(see_in_dark, 8)
-		lighting_alpha = min(lighting_alpha, LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE)
```

Comment:
Shouldn't actually impact anything; there's almost nothing that has x-ray vision that's a carbon subtype that also doesn't have an override.